jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -95,22 +95,26 @@
         </div>
       </v-row>
 
-      <v-row v-for="(item, index2) in region" class="row-no-outside-padding">
-        <v-text-field v-if="item.name === 'Parent' || item.name === 'Masked values'"
-          :label="item.name"
-          :value="item.value"
-          style="padding-top: 0px; margin-top: 0px"
-          :readonly="true"
-          :hint="item.name === 'Parent' ? 'Subset was defined with respect to this reference data (read-only)' : 'Number of elements included by mask'"
-        ></v-text-field>
-        <v-text-field v-if="item.name !== 'Parent' && item.name !== 'Masked values'"
-          :label="item.name"
-          v-model.number="item.value"
-          type="number"
-          style="padding-top: 0px; margin-top: 0px"
-          :suffix="item.unit ? item.unit.replace('Angstrom', 'A') : ''"
-        ></v-text-field>
-      </v-row>
+      <div v-for="(item, index2) in region">
+        <v-row v-if="item.name === 'Parent' || item.name === 'Masked values'" class="row-no-outside-padding">
+          <v-text-field
+            :label="item.name"
+            :value="item.value"
+            style="padding-top: 0px; margin-top: 0px; margin-bottom: 10px;"
+            :readonly="true"
+            :hint="item.name === 'Parent' ? 'Subset7 was defined with respect to this reference data (read-only)' : 'Number of elements included by mask'"
```

Comment:
```suggestion
            :hint="item.name === 'Parent' ? 'Subset was defined with respect to this reference data (read-only)' : 'Number of elements included by mask'"
```

Reply:
Oh, that was accidentally included while I was testing. I removed that in the following commit.